olap/src/aggregator/mod.rs

```diff
@@ -94,7 +107,7 @@ pub async fn run(
                     true,
                 )),
                 Arc::new(Field::new("event_cid", DataType::Binary, false)),
-                Arc::new(Field::new("state", DataType::Utf8, false)),
+                Arc::new(Field::new("state", DataType::Utf8, true)),
```

Comment:
Why are we changing the nullable to be true here?

Reply:
Ran into this with some real world bad data. If the events are broken this allows us to continue to process other events and just use null as the state for the broken ones.